courses/free-courses-ko.md

```diff
@@ -4,7 +4,7 @@
 * [Android](#android)
 * [Arduino](#arduino)
 * [ASP.NET](#asp.net)
-* [C/C++](#c/c++)
```

Comment:
good catch - however, I think the best thing to do is to observe that the id that the markdown processor creates for this heading is just #cc (it removes characters not allowed in html ids and changes to lower case.)

Reply:
Ok 😃

Reply:
Reverted in f7896fab0f35ec244ebb5b7168f93e163b6745fc

Reply:
> Reverted in [f7896fa](https://github.com/EbookFoundation/free-programming-books/commit/f7896fab0f35ec244ebb5b7168f93e163b6745fc)

Ok

Reply:
Resolved at #7029 with commit 376569a55dc09ea77569298d4ffa88c7b9ee39e9